app/server/appsmith-plugins/mssqlPlugin/src/main/resources/setting.json

```
@@ -5,21 +5,66 @@
+              "value": "MANUAL"
+            },
+            {
+              "label": "On page load",
+              "value": "ON_PAGE_LOAD"
+            },
+            {
+              "label": "Automatic (when used)",
+              "value": "AUTOMATIC"
+            }
+          ],
+          "initialValue": "MANUAL",
```

Biome (1.9.4)

[error] The key initialValue was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)